gems/pending/spec/appliance_console/database_replication_standby_spec.rb

```diff
@@ -58,81 +58,29 @@
+      subject.run_repmgrd_configuration = false
       expect(subject).to receive(:data_dir_empty?).and_return(true)
       expect(subject).to receive(:generate_cluster_name).and_return(true)
       expect(subject).to receive(:create_config_file).and_return(true)
       expect(subject).to receive(:clone_standby_server).and_return(true)
       expect(subject).to receive(:start_postgres).and_return(true)
       expect(subject).to receive(:register_standby_server).and_return(true)
-      expect(subject).to receive(:configure_repmgrd).and_return(true)
+      expect(subject).to receive(:write_pgpass_file).and_return(true)
       expect(subject.activate).to be true
     end
 
-    it "returns false when data_dir_empty? fails" do
-      expect(subject).to receive(:data_dir_empty?).and_return(false)
-      expect(subject).to_not receive(:generate_cluster_name)
-      expect(subject).to_not receive(:create_config_file)
-      expect(subject).to_not receive(:clone_standby_server)
-      expect(subject).to_not receive(:start_postgres)
-      expect(subject).to_not receive(:register_standby_server)
-      expect(subject).to_not receive(:configure_repmgrd)
-      expect(subject.activate).to be false
-    end
-
-    it "returns false when generate_cluster_name fails" do
-      expect(subject).to receive(:data_dir_empty?).and_return(true)
-      expect(subject).to receive(:generate_cluster_name).and_return(false)
-      expect(subject).to_not receive(:create_config_file)
-      expect(subject).to_not receive(:clone_standby_server)
-      expect(subject).to_not receive(:start_postgres)
-      expect(subject).to_not receive(:register_standby_server)
-      expect(subject).to_not receive(:configure_repmgrd)
-      expect(subject.activate).to be false
-    end
-
-    it "returns false when create_config_file fails" do
-      expect(subject).to receive(:data_dir_empty?).and_return(true)
-      expect(subject).to receive(:generate_cluster_name).and_return(true)
-      expect(subject).to receive(:create_config_file).and_return(false)
-      expect(subject).to_not receive(:clone_standby_server)
-      expect(subject).to_not receive(:start_postgres)
-      expect(subject).to_not receive(:register_standby_server)
-      expect(subject).to_not receive(:configure_repmgrd)
-      expect(subject.activate).to be false
-    end
-
-    it "returns false when clone_standby_server fails" do
-      expect(subject).to receive(:data_dir_empty?).and_return(true)
-      expect(subject).to receive(:generate_cluster_name).and_return(true)
-      expect(subject).to receive(:create_config_file).and_return(true)
-      expect(subject).to receive(:clone_standby_server).and_return(false)
-      expect(subject).to_not receive(:start_postgres)
-      expect(subject).to_not receive(:register_standby_server)
-      expect(subject).to_not receive(:configure_repmgrd)
-      expect(subject.activate).to be false
-    end
-
-    it "returns false when start_postgres fails" do
```

Comment:
@carbonin Please describe why you are removing these tests. Don't they help with code coverage?